OpenRA.Mods.Common/Scripting/Properties/MobileProperties.cs

```diff
@@ -31,20 +32,24 @@ public MobileProperties(ScriptContext context, Actor self)
 			"(in cells) that will be considered close enough to complete the activity.")]
 		public void Move(CPos cell, int closeEnough = 0)
 		{
-			Self.QueueActivity(new Move(Self, cell, WDist.FromCells(closeEnough)));
+			if (!mobile.IsTraitDisabled)
+				Self.QueueActivity(new Move(Self, cell, WDist.FromCells(closeEnough)));
```

Comment:
I think this would be better left unchanged. Since this is a queued activity, the trait might not actually be disabled anymore when the activity starts to run. It also already does its own `IsTraitDisabled` check, so checking here is redundant.